types/block.go

```diff
@@ -89,7 +89,7 @@ func (dah *DataAvailabilityHeader) Hash() []byte {
 	for i, rowRoot := range dah.RowsRoots {
 		slices[i] = rowRoot.Bytes()
 	}
-	for i, colRoot := range dah.RowsRoots {
+	for i, colRoot := range dah.ColumnRoots {
```

Comment:
Ouch 🙈 a classic bug that would have been caught with some simple unit tests or a proper review! 

Thanks a lot for this @evan-forbes!